mw-server/main.go

```diff
@@ -129,6 +134,11 @@ func init() {
 
 	fmt.Println("PostgreSql connected successfully...")
 
+	geminiClient, err = genai.NewClient(ctx, option.WithAPIKey(config.Env.GeminiApiKey))
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
```

Comment:
let's checl better for env type (prod vs dev instead of checking variables itself)

Reply:
Yes, I will add env type check, although there is no variable check or any other check

Reply:
Done